templates/document.html.slim

```diff
@@ -253,16 +259,30 @@ html lang=(attr :lang, 'en' unless attr? :nolang)
         // For slides that do not fit on a page, max number of pages
         pdfMaxPagesPerSlide: #{attr 'revealjs_pdfmaxpagesperslide', 1},
 
+        //default loaded plugin register
+        //plugins: [ RevealHighlight, RevealNotes,RevealZoom ],
+        plugins: [ #{revealjs_plugins(document,node,revealjsdir)} ],
+
         // Optional libraries used to extend on reveal.js
         dependencies: [
             #{revealjs_dependencies(document, self, revealjsdir)}
         ],
 
         #{(attr? 'revealjs_plugins_configuration') ? File.read(attr('revealjs_plugins_configuration', '')) : ""}
 
-      });
+      };
+
+    /other plugins use js file to loading
+    - if attr? :revealjs_config
+      script src=(attr 'revealjs_config', '') 
```

Comment:
We should use docinfo to enable additional plugins.

Reply:
I just mimic the code that has been in the repo. After checking the docinfo mechanism, It is much more straightforward. So I have updated to use docinfo to load additional plugins.